frappe/printing/page/print/print.js

```diff
@@ -657,7 +657,7 @@ frappe.ui.form.PrintView = class {
 	}
 
 	get_letterhead() {
-		return this.letterhead_selector.val();
+		return this.letterhead_selector.val() || "No Letterhead";
```

Comment:
```suggestion
		return this.letterhead_selector.val() || __("No Letterhead");
```

Reply:
ah i don't think i have push rights to your repo - not able to see the commit changes button :sweat_smile: